data_store/cli.py

```diff
@@ -18,148 +23,292 @@
+            webbrowser.open(presigned_url, new=0, autoraise=True)
+
+
+@admin_cli.command("reingest-file")
+@click.argument("filepath", required=True, type=click.Path(exists=True, dir_okay=False, file_okay=True))
+@click.argument("submission_id", required=True, type=str)
+def reingest_local_single_file(filepath, submission_id):
+    """Reingest a locally-saved submission file.
+
+    :param filepath (str):  Path to a submission file to be re-ingested
+    :param submission_id (str):  String of the human readable submission ID (eg. S-PF-R01-1) being reingested
+
+    Example usage:
+        flask admin reingest-file <filepath> <submission_id>
+    """
+    with current_app.app_context():
+        print(f"Reingesting submission {submission_id} from {filepath}.")
+        reingest_file(filepath, submission_id)
+
+
+@admin_cli.command("reingest-s3")
+@click.argument("filepath", required=True, type=click.Path(exists=True, dir_okay=False, file_okay=True))
+def reingest_files_from_s3(filepath):
+    """Reingest files from the 'sucessful files' S3.
+
+    :param filepath (str):  Path to a file containing line-separated submission IDs to be re-ingested
+
+    Example usage:
+        flask admin reingest-s3 <filepath>
+    """
+
+    with current_app.app_context():
+        with click.open_file(filepath) as file:
+            reingest_outputs = reingest_files(file)
+            if False in reingest_outputs["Success"].values:
+                print("Some submissions failed to re-ingest. Please see the output for details.")
+            else:
+                print("All submissions re-ingested successfully.")
+            pd.set_option("display.max_colwidth", None)
+            print(reingest_outputs)
+
+
+@admin_cli.command("set-roles-to-users")
+@click.option("--filepath", required=True, type=click.Path(exists=True, dir_okay=False, file_okay=True))
+@click.option("--roles", type=str, help="List of roles comma separated to assign to each account")
+def set_roles_to_users(filepath, roles):
+    """Assign roles to the users from the csv.
+
+    :param filepath:  Path to csv file containing email, full name and azure subject id
+    :param roles (list):  List of roles to assign to each comma separated
+
+    Example usage:
+        flask admin set-roles-to-users --filepath <path> --roles <role1>,<role2>
+    """
+
+    result = {
+        "created": 0,
+        "updated": 0,
+        "errors": [],
+    }
+
+    roles = roles.split(",") if roles else []
+
+    click.secho(f"\nSetting roles to users from {filepath}", fg="green")
+    click.secho(f"Assigning roles: {roles}", fg="green")
+    click.secho("\nDetails:", fg="blue")
+
+    with current_app.app_context():
+        with click.open_file(filepath) as csvfile:
+            reader = csv.DictReader(csvfile)
+            for row in reader:
+                email = row["email"]
+                full_name = row["full_name"]
+                azure_ad_subject_id = row["azure_ad_subject_id"]
```

Comment:
Should we default this if we don't care about its value most of the time?

Reply:
Azure_ad_subsject_id defaults to `uuid4`, if not provided.
full_name defaults to empty string.